tests/bench/locomo.py

```
@@ -0,0 +1,909 @@
+                response = await client.post(
+                    url,
+                    json=payload,
+                    timeout=30.0,
+                )
+                if response.status_code != 204:
+                    print(
+                        f"[{workspace_id}] ERROR: Dream trigger failed with status {response.status_code}"
+                    )
+                    print(f"[{workspace_id}] Response body: {response.text}")
+                    return False
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)